backend/routes/cronJob.js

```diff
@@ -72,6 +75,14 @@ async function createPlayer(player) {
     ? "0"
     : player.effect_fg_percent;
 
+  const performanceScores = await calcPerformanceScores(player);
+  player.outsideOffenseScore = performanceScores.outsideOffenseScore;
+  player.insideOffenseScore = performanceScores.insideOffenseScore;
+  player.offenseDisciplineScore = performanceScores.offenseDisciplineScore;
+  player.defenseDisciplineScore = performanceScores.defenseDisciplineScore;
+  player.consistencyScore = performanceScores.consistencyScore;
+  player.reboundingScore = performanceScores.reboundingScore;
+
```

Comment:
No need to pull all the metrics out into variable right? You can just use performanceScores directly below

Reply:
Very true. I already got rid of it with your suggestion to clean up the calls to update the player data.